subprojects/logging/src/main/java/org/gradle/internal/logging/console/BuildStatusRenderer.java

```diff
@@ -94,10 +99,16 @@ private String trimToConsole(String str) {
         return str;
     }
 
-    private void renderNow() {
-        if (currentBuildStatus != null) {
-            buildStatusLabel.setText(Arrays.asList(new Span(Style.of(Style.Emphasis.BOLD), trimToConsole(currentBuildStatus))));
+    private void renderNow(long now) {
+        if (currentBuildStatus != null && !currentBuildStatus.isEmpty()) {
+            String elapsedTime = elapsedTimeFormatter.format(now - buildStartTimestamp);
+            buildStatusLabel.setText(Arrays.asList(
+                new Span(Style.of(Style.Emphasis.BOLD), trimToConsole(format(currentBuildStatus, elapsedTime)))));
         }
         console.flush();
     }
+
+    private static String format(String status, String elapsedTime) {
+        return status + "... " + elapsedTime;
```

Comment:
I kinda preferred what you showed before with the [1.5s] instead of going full-Buck with ... 1.5s